scrapers/CorbinFisher.yml

```diff
@@ -27,4 +34,23 @@ xPathScrapers:
       Studio:
         Name:
           fixed: CorbinFisher
-# Last Updated July 21, 2021
+
+  performerScraper:
+    performer:
+      Name: //div[@class="modelBioDetails"]//h2/text()
+      Gender:
+        fixed: Male
+      HairColor: //span[contains(text(),"HAIR COLOR:")]/following-sibling::text()
+      EyeColor: //span[contains(text(),"EYES:")]/following-sibling::text()
+      Weight:
+        selector: //span[contains(text(),"WEIGHT:")]/following-sibling::text()
+        postProcess:
+          - lbToKg: true
+      Height:
+        selector: //span[contains(text(),"HEIGHT:")]/following-sibling::text()
+        postProcess:
+          - feetToCm: true
+      Measurements: //span[contains(text(),"COCK SIZE:")]/following-sibling::text()
```

Comment:
If you dont care about .5 precision you can use the below
```yaml
        selector: //span[contains(text(),"COCK SIZE:")]/following-sibling::text()
        postProcess:
          - replace:
              - regex: ^
                with: "0'"
          - feetToCm: true
```
to have the size in cm
With the above  5 = 5.5 = 5.xx inches -> 13 cm. This is mostly a hack though so not sure if you want it